oppai.c

```diff
@@ -977,6 +978,12 @@ int p_objects(ezpp_t ez, slice_t* line) {
   }
 
   o->time = p_float(&e[2]);
+  if (o->time == infinity) {
```

Comment:
please use is_inf from main.c (you can move it to oppai.c for now)

https://github.com/Francesco149/oppai-ng/blob/43b8fe89c31833e6d8a06bd6bd786092def5a4d4/main.c#L362

comparing to infinity is not reliable due to how floating point works